Gemfile

```diff
@@ -53,9 +53,7 @@ group :development, :test do
   gem 'toxiproxy', '~> 0.1.3'
 end
 
-group :development do
-  gem 'rails-erd'
-end
+gem 'rails-erd', group: :development
```

Comment:
Since this PR is changing 169 files I think it will be nice to keep it as minimal as it is possible for future reviewers.